### DISCUSSION TITLE
Docker and rclone.

### DISCUSSION THREAD

**Discussion by AyaanZaveri** [coder/code-server#3974] [Q&A]
Is there something I can add to the Dockerfile to get Docker CLI. I tried installing it, but it wouldn't work. I'm running it code-server on Railway. Also, is there a way to mount rclone? I keep running into fuse errors that I can't fix.

Thanks!

**Comment by jsjoeio** [comment]
Good question! Let's see what @bpmct knows (note: afterwards, this would be good for us to document for future users).

_Side note: in the future, feel free to use the Q&A Discussions for these types of questions instead of issues _

**Reply by AyaanZaveri** [reply]
Thanks, will do!
